File: backend/components/metiers/threed/rgpd/helpers/rgpd_helper.py
# rgpd_helper.py – Helper RGPD (Python)
def mask_pii(data):
    """Masque les informations personnelles identifiables (PII) dans les données."""
    masked = {}
    for key, value in data.items():
        if key.lower() in ("nom", "prenom", "email", "telephone", "adresse", "id", "identifiant", "ssn", "nss", "num_secu"):
            masked[key] = "***"
        else:
            masked[key] = value
    masked["masked"] = True
    return masked

File: backend/components/metiers/threed/rgpd/helpers/rgpd_helper.js
// rgpd_helper.js – Helper RGPD (JS)
/**
 * Exemple de helper RGPD
 */
function maskPII(data) {
  if (data === null || data === undefined) {
    throw new Error('Data cannot be null or undefined');
  }
  return { ...data, masked: true };
}

module.exports = { maskPII };
